features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -0,0 +1,81 @@
+Feature: cluster monitoring related upgrade check
+  # @author hongyli@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: upgrade cluster monitoring along with OCP - prepare
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-monitoring" project
```

Comment:
Line 7 is no need

Reply:
It's needed, the prepare has no effect for the upgrade check. Prepare run on the old release, check run in the upgraded release and the two parts run in different v3 jobs